src/Settings/components/comon/bottomSheetComponent.jsx

```diff
@@ -9,18 +10,33 @@ function BottomSheetComponent({ isVisible, actionConstant, value, title, action,
   const isNightMode = useSelector((state) => state.isNightMode);
   const { containerNightStyles, textNightStyle } = nightModeStyles(isNightMode);
   return (
-    <BottomSheet modalProps={{}} isVisible={isVisible} onBackdropPress={() => toggleVisible(false)}>
-      <Text style={[styles.bottomSheetTitle, textNightStyle, containerNightStyles]}>{title}</Text>
-      <Divider />
-      {actionConstant.map((item) => (
-        <RenderBottomSheetItem
-          key={item.key}
-          item={item}
-          toggleVisible={toggleVisible}
-          value={value}
-          action={action}
-        />
-      ))}
+    <BottomSheet
+      modalProps={{
+        supportedOrientations: [
+          "portrait",
+          "portrait-upside-down",
+          "landscape-left",
+          "landscape-right",
+        ],
+      }}
+      isVisible={isVisible}
+      onBackdropPress={() => toggleVisible(false)}
+    >
+      <View
+        style={{ width: "90%", marginLeft: "auto", marginRight: "auto", bottom: 0, height: "90%" }}
```

Comment:
Have we used inline styles anywhere else in code, I couldn't find it in this PR diff. If there is a separate place for styles, can we move this style in that file as well?